rozk04-sp2

Katerina-33 opened the pull request:
Link to application: https://eso.vse.cz/~rozk04/sp2/

Katerina-33 committed:
rozk04-sp2

Link to application: https://eso.vse.cz/~rozk04/sp2/

Katerina-33 committed:
version 02

changed almost everything, created the whole app

nvbach91 commented:
- ovladaci prvky formulare by mely byt v elementu form + udalost submit
- zmensit postery (+pouzit fixni velikost) aby se jich tam veslo vice + aby byl videt popis
- movie description zkratit a pridat za to tri tecky, nebo dat do scrollovatelneho containeru
- nesmi se pouzivat inline javascript
- nesmi se pouzivat inline css
- nepouzivat onclick, onsubmit, onload, ...
- chybi indikace nacitani dat
- nepouzivat `alert()`
- pokud film uz je v oblibenych, zakazat tlacitko "pridat do oblibenych", nebo misto toho zobrazit tlacitko "odebrat z oblibenych"
- do localstorage ukladat vcetne id filmu a pote pridat tlacitko zobrazit vice informaci o filmu do seznamu posledni vyhledavani a do seznamu oblibenych
- nesmi se zasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, zmena obsahu elementu, ...)
- zobrazit take zanry filmu https://developer.themoviedb.org/reference/genre-movie-list

nvbach91 commented on www/ROZK04/sp2/assets/js/script.js:
```diff
@@ -0,0 +1,285 @@
+async function showMovieDetails(movieId) {
+    try {
+        // calling API
+        const movieDetailsUrl = `https://api.themoviedb.org/3/movie/${movieId}`;
```
cast stringu `https://api.themoviedb.org` dat do konstanty

Katerina-33 committed:
version 03

oprava na základě feedbacku, omlouvám se za pozdní odevzdání k zítřejší obhajobě

Katerina-33 committed:
Version 04

Úprava - obhajoba
